app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class VotesController < ApplicationController
+  def create
```

Comment:
This isn't being called anywhere so it would be cleaner to delete it.
If it isn't clear that `Vote.create()` doesn't call this method, that's definitely something worth chatting with a classmate, instructor, or TA about to get clarity on.